File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/responses/GetWalletsResponse.java
package com.elgin.elginexperience.shipay.responses;

import com.elgin.elginexperience.shipay.models.wallet.Wallet;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GetWalletsResponse {
    @SerializedName("wallets")
    public List<Wallet> wallets;
}

File: M8/Exemplos/App_eXperience_Kotlin/app/src/main/java/com/example/e1_elgin_kotlin/shipay/responses/GetWalletsResponse.kt
package com.example.e1_elgin_kotlin.shipay.responses

import com.example.e1_elgin_kotlin.shipay.models.wallet.Wallet
import com.google.gson.annotations.SerializedName

data class GetWalletsResponse (
    @SerializedName("wallets")
    var wallets: List<Wallet>
)
